libs/ai-endpoints/langchain_nvidia_ai_endpoints/reranking.py

```diff
@@ -71,14 +71,14 @@ def __init__(self, **kwargs: Any):
         super().__init__(**kwargs)
         self._client = _NVIDIAClient(
             base_url=self.base_url,
-            model=self.model,
-            default_model=self._default_model_name,
+            model_name=self.model,
+            default_hosted_model_name=self._default_model_name,
```

Comment:
I guess only reranker has _default_model_name, and others have _default_name. Shall we make that consistant

Reply:
good idea, i'll make them all _default_model_name